scripts/kconfig/kconfigfunctions.py

```
@@ -752,7 +752,7 @@ def dt_compat_on_bus(kconf, _, compat, bus):
 def dt_compat_any_has_prop(kconf, _, compat, prop):
```

perhaps extend the existing function to take an optional value argument.

Like this:
```suggestion
def dt_compat_any_has_prop(kconf, _, compat, prop, value=None):
```

and then adjust the flow according based on `value is None`.

---

okay, that makes sense, I'll update the PR for this suggestion

---

any updates ?